server/app/views/style/BaseStyles.java

```diff
@@ -30,6 +30,23 @@ public final class BaseStyles {
   public static final String BG_CIVIFORM_PURPLE_LIGHT = "bg-civiform-purple-light";
   public static final String TEXT_CIVIFORM_YELLOW = "text-civiform-yellow";
   public static final String BG_CIVIFORM_YELLOW_LIGHT = "bg-civiform-yellow-light";
+  public static final String BG_CIVIFORM_TEAL = "bg-civiform-teal";
+
+  /////////////////////////////////////////////////////////////////////////////////////////////////
+  // USWDS component classes
```

Comment:
question (non-blocking):  I'm pondering whether there's an advantage to adding the USWDS styles here.  Seems like the class names are already clear.

Reply:
I don't really understand that comment because we use plenty of styles that aren't in `BaseStyles.java` and, as far as I can tell, they aren't stripped out in production.  Regardless, these USWDS styles won't come from the `tailwind.css` stylesheet, they'll be coming from the separate `uswds.min.css` file.